docs/public/using/backend/rpcapi-reference.md

```diff
@@ -1248,11 +1248,14 @@ This method is not available if [`RPCAPI.enable_add_batch_job`][RPCAPI.enable_ad
 is disabled (enabled by default).
 
 A [*username*][Username] and its [*api key*][Api key] can be added with the
-[`add_api_user`][API add_api_user] method. A [*username*][Username] can only have
-one un-finished *batch* at a time.
+[`add_api_user`][API add_api_user] method.
 
 *Tests* enqueud using this method are assigned a [*priority*][Priority] of 5.
 
+> In previous versions of Zonemaster-Backend a new batch could not be created by
+> the same [*username*][Username] if that *username* had created a batch that was
+> not yet finished. That restriction has been removed.
```

Comment:
I think notes on how this version differs from the previous version belong in the release notes, not in the reference documentation for this version. Don't you agree?

Reply:
There should of course be a comment in the release notes, but sometimes I think such note can be reasonable in the documentation, not forever, but for a while. That is why I added it.

Reply:
I agree with @matsduf here: it isn’t unusual to state in the documentation that some feature X is available from version Y onwards. A comment in the release notes will most likely be read by administrators of existing Zonemaster installations who are upgrading to a later versions; however, administrators of brand new installations might not think about reading release notes.

I just suggest adding the version number (e.g. “That restriction has been removed in version 2023.2”.)

Reply:
Updated as suggested with version number.